hardhat.config.js

```diff
@@ -3,6 +3,7 @@ require("solidity-coverage");
 require("hardhat-deploy");
 require("@ethersproject/constants");
 require("@gnosis.pm/safe-contracts");
+require("hardhat-gas-reporter");
 
```

Comment:
You might want to use this one as well:
`require("hardhat-contract-sizer");`

and then:
`npx hardhat size-contracts`

Here's the link for configs, if needed:
https://www.npmjs.com/package/hardhat-contract-sizer